apps/src/util/BrowserTextToSpeech.ts

```diff
@@ -0,0 +1,42 @@
+import markdownToTxt from 'markdown-to-txt';
+
+import currentLocale from './currentLocale';
+
+/**
+ * Manages native Browser Text to Speech functionality.
+ */
+
+let ttsAvailable = false;
+
+speechSynthesis.addEventListener('voiceschanged', () => {
+  ttsAvailable = speechSynthesis.getVoices().length > 0;
```

Comment:
for my understanding: is `speechSynthesis` a global variable?

Reply:
Yep, it's available on the browser `window`: https://developer.mozilla.org/en-US/docs/Web/API/SpeechSynthesis